lib/bokken/documents.ex

```diff
@@ -117,4 +121,11 @@ defmodule Bokken.Documents do
   def change_file(%File{} = file, attrs \\ %{}) do
     File.changeset(file, attrs)
   end
+
+  def verify_total_size(file, user_id) do
+    user = Accounts.get_user!(user_id) |> Repo.preload(:files)
+
+    total_size = Enum.reduce(user.files, 0, fn file, acc -> acc + File.file_size(file) end)
+    total_size + File.file_size(file)
+  end
 end
```

Comment:
This should be a column in the user table. This way, you don't have to perform a full table scan on the files table whenever a user uploads a new file

Reply:
Considering that the size and quantity of files per user are not very large, does it make sense to store the size in the database? Performing that function for each user is extremely lightweight, while putting it in the database can generate other issues.